activesupport/lib/active_support/core_ext/string/output_safety.rb

```diff
@@ -136,15 +136,39 @@ def encode_with(coder)
 
     UNSAFE_STRING_METHODS.each do |unsafe_method|
       if unsafe_method.respond_to?(unsafe_method)
+        arity = unsafe_method.method(unsafe_method).arity
```

Comment:
`unsafe_method.respond_to?` and `unsafe_method.method` made me look twice.  What do you think about changing these to `""`, for clarity?

```suggestion
      if "".respond_to?(unsafe_method)
        arity = "".method(unsafe_method).arity
```

We use `# frozen_string_literal: true`, so it shouldn't cause an extra allocation.

Reply:
I have no preference about that, so I just followed the current code style. The world has actually changed since the moment of 735abe93a5590f3d6f215a645e633d9fcd8ff3b6, so we could revert this change back to be `""` as you suggest, but that coding style cleanup is a different concern from what I'm doing in this commit.